pkg/txutils/auth.go

```diff
@@ -22,6 +23,7 @@ import (
 //
 // expect tx.Unsigned type to be in [txs.AddSubnetValidatorTx, txs.CreateChainTx]
 func GetAuthSigners(tx *txs.Tx, network models.Network, subnetID ids.ID) ([]string, error) {
+	time.Sleep(2 * time.Second)
```

Comment:
add TODO? remove this when the issue is solved (most probably due to https://github.com/ava-labs/avalanche-cli/issues/775)